### ISSUE TITLE
After Updating Dio 3.0.7  getting error in FormData the method 'form' isn't defined for class FormData..

### ISSUE THREAD

**Issue by Zeeshan0201** [flutterchina/dio#591]
![Screenshot 2019-12-05 at 6 25 56 PM](https://user-images.githubusercontent.com/45591301/70237518-6a54ee80-178d-11ea-8e8a-9461b4c7216f.png)
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |      |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |      |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by sudocvua** [comment]
I get the same issue.
Does someone solve?

**Comment by mrcs13** [comment]
I get the same issue.

**Comment by deepakkj6** [comment]
I am facing the same issue! Anyone has any fix?

**Comment by deepakkj6** [comment]
Try ` FormData formData = FormData.fromMap(<String, dynamic>{
    "name": "wendux",
  });`

**Comment by abhikrnigam** [comment]
I am having the same issue. Updated to dio:3.0.9 and got this error. Also if downgrade to 2.2.2 parse function stops working and build is failed.

**Comment by mustafabulutt** [comment]
> Deneyin ` FormData formData = FormData.fromMap(<String, dynamic>{ "name": "wendux", });`

nice bro you code work :)

**Comment by khaconoc** [comment]
if you user GetX then you try ` import 'package:get/get.dart' hide Response, FormData, MultipartFile;`

**Comment by FatimaNamm9** [comment]
I have same issue ,solve it by add   import 'package:dio/dio.dart'; with prefix 
problem was because I also added import 'package:get/get.dart'; 
so
The name 'FormData' is defined in the libraries 'package:dio/src/form_data.dart (via package:dio/dio.dart)' and 'package:get/get_connect/http/src/multipart/form_data.dart'.

**Comment by ShafiMunshi** [comment]
I had the same issue when I worked with getx. so I used the FormData like this, 
1. first import the dio package like this prefix 
import 'package:dio/dio.dart' as dioo;

2. then use 
 dioo.FormData formData = dioo.FormData.fromMap({'id': mobileNumber, 'pass': password});
